packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -180,15 +180,13 @@ function renderStyles<T extends ElementType = 'button'>(props: SpectrumButtonPro
           onPress={action('press')}
           onPressStart={action('pressstart')}
           onPressEnd={action('pressend')}
-          variant="primary"  
```

Comment:
Did you remove the primary because all the stories using this and the ones before define their `variant` and we don't need a default?